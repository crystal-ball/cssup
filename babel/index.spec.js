// src/__tests__/index-test.js
const babel = require('babel-core')
const plugin = require('./index')

var example = `
const regularVariable = 'test'

const testStyles = cssup\`
.test {
  color: 'blue';
}\`

const skipTemplateString = trick\`don't interact\`
`

it('works', () => {
  const { code } = babel.transform(example, { plugins: [plugin] })
  expect(code).toMatchSnapshot()
})
