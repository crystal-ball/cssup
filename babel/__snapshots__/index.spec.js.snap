// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`works 1`] = `
"
const regularVariable = 'test';

const testStyles = cssup\`
.test {
  color: 'blue';
}\`;

const skipTemplateString = trick\`don't interact\`;"
`;
